CMakeLists.txt

```diff
@@ -2930,29 +2931,30 @@ listtostr(EXTRA_CFLAGS _EXTRA_CFLAGS)
 set(EXTRA_CFLAGS ${_EXTRA_CFLAGS})
 
 # Compat helpers for the configuration files
+
+if(EXISTS "${CMAKE_SOURCE_DIR}/VERSION")
+  file(READ "${CMAKE_SOURCE_DIR}/VERSION" SDL_SOURCE_VERSION)
```

Comment:
```suggestion
if(EXISTS "${PROJECT_SOURCE_DIR}/VERSION")
  file(READ "${PROJECT_SOURCE_DIR}/VERSION" SDL_SOURCE_VERSION)
```

Reply:
Good thinking. This is presumably the same reasoning as in <https://gitlab.freedesktop.org/dbus/dbus/-/merge_requests/332>?

There are a few pre-existing uses of `CMAKE_SOURCE_DIR` which should probably also be `PROJECT_SOURCE_DIR`, but I think that should be an orthogonal change.

Reply:
> There are a few pre-existing uses of CMAKE_SOURCE_DIR which should probably also be PROJECT_SOURCE_DIR, but I think that should be an orthogonal change.

Actually, the only one that wants to be PROJECT_SOURCE_DIR is when we're running `git rev-list` (before) or `git describe` (now), so I changed that too.

Reply:
Yes, good thinking about changing the working directory of `git` too.
That is important when projects use SDL as a submodule.